api-catalog-ui/frontend/package.json

```diff
@@ -8,16 +8,45 @@
         "@emotion/core": "10.1.1",
         "@emotion/styled": "10.0.27",
         "agentkeepalive": "4.5.0",
+        "babel-plugin-syntax-jsx": "^6.18.0",
         "buffer": "6.0.3",
+        "caseless": "^0.12.0",
+        "cookie-signature": "^1.2.1",
+        "core-js": "^3.37.1",
+        "css.escape": "^1.5.1",
+        "dashdash": "^2.0.0",
+        "deep-diff": "^1.0.2",
+        "ee-first": "^1.1.1",
```

Comment:
why do you need to add dependencies that were not here before? These are most likely transitive

Reply:
Also, please don't use the caret `^` in dependencies, this allows for automatic minor updates which could cause issues

Reply:
> why do you need to add dependencies that were not here before? These are most likely transitive

they were reported vulnerable in BD scan

Reply:
I guess the workflow should be to first try to upgrade the top-direct dependency that brings these transitive ones, then if that's not possible, an option is to override the ones transitive in the overrides section.